source/solutions-library/card-fraud-solution.txt

```diff
@@ -0,0 +1,220 @@
+integrates Apache Spark and MongoDB. Apache Spark, hosted by Databricks, allows the
+processing and analysis of large amounts of data in real-time. The Spark Connector
+translates MongoDB data into Spark data frames and supports real-time Spark streaming.
+
+The `App Services features <https://www.mongodb.com/atlas/app-services>`__ offered by
+MongoDB allow for real-time processing of data through change streams and triggers.
+Because |service-fullname| is capable of storing and processing various types of data as well
+as streaming capabilities and trigger functionality, it is well suited for use in an
+event-driven architecture.
+
+This solution uses the rich connector ecosystem of :atlas:`MongoDB and App Services
+</app-services/triggers/>` to process transactions in
+real-time. The App Service Trigger function is used by invoking a REST service call to an
+AI/ML model hosted through the Databricks MLflow framework.
+
+The example solution manages rules-based fraud prevention by storing user-defined payment
+limits and information in a user settings collection, as shown. This includes maximum
+dollar limits per transaction, the number of transactions allowed per day, and other
+user-related details. By filtering transactions based on these rules before invoking
+expensive AI/ML models, the overall cost of fraud prevention is reduced.
+
+Databricks as an AI/ML Ops Platform
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Databricks is a powerful AI/ML platform to develop models for identifying fraudulent
+transactions. One of the key features of Databricks is the support of real-time analytics.
+As discussed above, real-time analytics is a key feature of modern fraud detection
+systems.
+
+Databricks includes MLFlow, a powerful tool for managing the end-to-end machine learning
+lifecycle. MLFlow allows users to track experiments, reproduce results, and deploy models
+at scale, making it easier to manage complex machine learning workflows. MLFlow offers
+model observability, which allows for easy tracking of model performance and debugging.
+This includes access to model metrics, logs, and other relevant data, which can be used to
+identify issues and improve the accuracy of the model over time. Additionally, these
+features can help in the design of modern fraud detection systems using AI/ML.
+
+Technologies and Products Used
+------------------------------
+
+MongoDB Modern, Multi-Cloud Database Platform
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `Atlas Clusters <https://www.mongodb.com/basics/clusters>`__
+- `Change Streams <https://www.mongodb.com/developer/products/mongodb/change-streams/>`__
+- `Atlas Triggers <https://www.mongodb.com/docs/atlas/app-services/triggers/>`__
+- `Spark Streaming Connector <https://www.mongodb.com/docs/spark-connector/v2.4/scala/streaming/>`__
+
+Partner Technologies
+~~~~~~~~~~~~~~~~~~~~
+- `Databricks <https://www.mongodb.com/partners/databricks>`__
+
+Key Learnings
```

Comment:
Oddly, the original article uses "Key Considerations" here. I'm fine with keeping Key Learnings for consistency, but wanted to make a note of it.